pulse-physiology_FHIR/ppe_fhir.ipynb

```diff
@@ -125,18 +173,74 @@
     "            print(\"Unable to load initial state file\")\n",
     "        else:\n",
     "            self.results = self.pulse.pull_data()\n",
+    "        self.data_management.save_csv(self.results)\n",
+    "\n",
+    "    # Advances the time by 1 second and records the patient's state after each action\n",
+    "    def advance_time_and_record(self, start, seconds, action=None):\n",
```

Comment:
The parameters start and seconds are not clear to me: what are the intended values?

Reply:
This was to show the interval of time, but a little modification is made with the new commit.